.github/workflows/test.yml

```diff
@@ -46,9 +54,18 @@ jobs:
           coverage: none
 
       - name: Install Composer dependencies
-        run: |
-          composer require "laravel/framework:${{ matrix.laravel }}" --no-interaction --no-update --ansi
-          composer update --prefer-${{ matrix.stability }} ${{ matrix.option }} --no-interaction --no-progress --ansi
+        run: composer require "laravel/framework:${{ matrix.laravel }}" --no-interaction --no-update --ansi
+
+      - name: Set minimum stability
+        if: ${{ matrix.unstable }}
+        run: composer config minimum-stability dev --ansi
+          
+      - name: Uninstall the package that is not compatible with Laravel 11
+        if: ${{ matrix.unstable }}
+        run: composer remove --dev vyuldashev/laravel-openapi --no-interaction --no-update
```

Comment:
Once https://github.com/vyuldashev/laravel-openapi/pull/109 is merged, this uninstallation will no longer be necessary.